src/index.html

```diff
@@ -11,7 +11,144 @@
+          >
+        </label>
+        <label class="field__label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field__label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field__label">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="field__label">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="field__label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="field__label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="field__label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cat"
+            value="yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cat"
+            value="no"
+          >
+          No
+        </label>
+        <label class="field__label">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="black"
+          >
+        </label>
+        <label class="field__label">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="goesToBed"
+            step="5"
+          >
+        </label>
+        <label class="field__label">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="field__label">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <label class="field__label">
+          Comments:
+          <textarea
+            name="comments"
+            cols="10"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="273" alt="image" src="https://user-images.githubusercontent.com/53611910/201547425-737c60e6-6336-4a8e-aee6-44ecb0184d9d.png">

Reply:
done